psi4/src/psi4/libcubeprop/cubeprop.cc

```diff
@@ -154,6 +158,52 @@ void CubeProperties::raw_compute_properties() {
             }
             if (indsa0.size()) compute_orbitals(Ca_, indsa0, labelsa, "Psi_a");
             if (indsb0.size()) compute_orbitals(Cb_, indsb0, labelsb, "Psi_b");
+        } else if (task == "FRONTIER_ORBITALS") {
+            std::vector<int> indsa0;
+            std::vector<int> indsb0;
+            std::vector<std::string> labelsa;
+            std::vector<std::string> labelsb;
+            CharacterTable ct = basisset_->molecule()->point_group()->char_table();
+            if (nalpha_ == nbeta_) {
+                indsa0.push_back(nalpha_-1);
+                labelsa.push_back(to_string(std::get<1>(info_a_[nalpha_-1]) + 1) + "-" +
+                                  ct.gamma(std::get<2>(info_a_[nalpha_-1])).symbol() + "_HOMO");
+                indsa0.push_back(nalpha_);
+                labelsa.push_back(to_string(std::get<1>(info_a_[nalpha_]) + 1) + "-" +
+                                  ct.gamma(std::get<2>(info_a_[nalpha_])).symbol() + "_LUMO");
+            } else {
+                indsa0.push_back(nalpha_);
+                labelsa.push_back(to_string(std::get<1>(info_a_[nalpha_]) + 1) + "-" +
+                                  ct.gamma(std::get<2>(info_a_[nalpha_])).symbol() + "_LUMO");
+                indsb0.push_back(nbeta_+1);
+                labelsb.push_back(to_string(std::get<1>(info_b_[nbeta_+1]) + 1) + "-" +
+                                  ct.gamma(std::get<2>(info_b_[nbeta_+1])).symbol() + "_LUMO");
+                indsa0.push_back(nalpha_-1);
+                labelsa.push_back(to_string(std::get<1>(info_a_[nalpha_-1]) + 1) + "-" +
+                                  ct.gamma(std::get<2>(info_a_[nalpha_-1])).symbol() + "_SOMO");
+                indsb0.push_back(nbeta_);
+                labelsb.push_back(to_string(std::get<1>(info_b_[nbeta_]) + 1) + "-" +
+                                  ct.gamma(std::get<2>(info_b_[nbeta_])).symbol() + "_SOMO");
+                indsa0.push_back(nalpha_-2);
+                labelsa.push_back(to_string(std::get<1>(info_a_[nalpha_-2]) + 1) + "-" +
+                                  ct.gamma(std::get<2>(info_a_[nalpha_-2])).symbol() + "_DOMO");
+                indsb0.push_back(nbeta_-1);
+                labelsb.push_back(to_string(std::get<1>(info_b_[nbeta_-1]) + 1) + "-" +
+                                  ct.gamma(std::get<2>(info_b_[nbeta_-1])).symbol() + "_DOMO");
+            }
+            if (indsa0.size()) compute_orbitals(Ca_, indsa0, labelsa, "Psi_a");
+            if (indsb0.size()) compute_orbitals(Cb_, indsb0, labelsb, "Psi_b");
+        } else if (task == "DUAL_DESCRIPTOR") {
```

Comment:
Add the DOI links here perhaps.

Reply:
I've added a short comment here and tried to describe the new additions in the docs.